src/stylesheets/layouts/_concept.scss

```diff
@@ -0,0 +1,380 @@
+        width: 155px;
+        top: -11%;
+      }
+    }
+
+    &__img6-bg {
+      width: 0;
+      @media (min-width: 800px) {
+        width: 300px;
+        align-self: center;
+        bottom: 0;
+      }
+    }
+
+    &__img6 {
+      align-self: center;
+      width: 260px;
+      top: -26.5%;
+      clip: rect(0px,250px,120px,0px);
+      @media (min-width: 800px) {
+        clip: rect(0px, 1000px, 1000px, 0px);
+        width: 250px;
+        top: 68%;
+      }
+    }
+
+    &__img7-bg {
+      align-self: center;
+      width: 280px;
+      top: 0;
+      transform: rotate(-180deg);
+      @media (min-width: 800px) {
+        align-self: center;
+        width: 400px;
+        top: 58.2%;
+        transform: rotate(360deg);
+      }
+    }
+
+    &__img7 {
+      align-self: center;
+      width: 120px;
+      top: -30%;
+      transform: rotate(-18deg);
+      @media (min-width: 800px) {
+        align-self: center;
+        height: 220px;
+        top: 63%;
+        transform: rotate(10deg);
+      }
+    }
+  }
+}
+
+.bubble {
+  &__wrapper {
+    display: flex;
+    flex-direction: row-reverse;
+  }
+
+  &__florence {
+    display: inline-block;
+    border-radius: 1.5rem;
+    background-color: $color-bubble-mint;
+    margin: 0.2rem 18% 0.2rem 0.2rem;
+    padding: 0.8rem 1.2rem;
+    @media (min-width: 800px) {
+      margin: 0.2rem 10% 0.2rem 0.2rem;
+    }
+
+    &--tail {
+      position: relative;
+      border-radius: 1.5rem;
+      background-color: $color-bubble-mint;
+      margin: 0.2rem 10% 0.2rem 0.2rem;
+      padding: 0.8rem 1.2rem;
+      @media (min-width: 800px) {
+        margin: 0.2rem 10% 0.2rem 0.2rem;
+      }
+
+      &:before {
+        content: "";
+        position: absolute;
+        z-index: -1;
+        bottom: -1px;
+        left: -7px;
+        height: 10px;
+        border-right: 30px solid $color-bubble-mint;
+        background: $color-bubble-mint;
+        border-bottom-right-radius: 150px 50px;
+        transform: translate(0, -2px) rotate(160deg) scaleY(-1);
+      }
+    }
+  }
+
+  &__patient {
+    position: relative;
+    border-radius: 1.5rem;
+    background-color: $color-light-blue;
+    margin: 0.2rem;
+    padding: 0.8rem 1.5rem;
+    flex-basis: content;
+    @media (min-width: 800px) {
+      padding: 0.8rem 1.2rem;
+    }
+
+    &:before {
+      content: "";
+      position: absolute;
+      z-index: -1;
+      bottom: -1px;
+      right: -7px;
+      height: 10px;
+      border-right: 30px solid $color-light-blue;
+      background: $color-light-blue;
+      border-bottom-right-radius: 150px 50px;
+      transform: translate(0, -2px) rotate(40deg);
+    }
+  }
+}
+
+.avatar {
+  width: 38px;
+  background-color: $color-yellow;
+  border-radius: 50%;
+  display: block;
+  margin-top: -38px;
+  margin-left: -44px;
+
+  &--1 {
+    margin-top: -75px;
+    margin-left: -44px;
+  }
+
+  &--2 {}
+
+  &--3 {}
+
+  &--4 {}
+
+  &--5 {}
+
+  &--6 {}
+
+  &--7 {}
```

Comment:
need these?

Reply:
I'm all for deleting anything that doesn't do anything and just adds to your CSS filesize